convert2rhel/toolopts.py

```diff
@@ -338,5 +388,52 @@ def warn_on_unsupported_options():
         utils.ask_to_continue()
 
 
+def options_from_config_files(cfg_path=None):
+    """Get the options from config file.
+
+    The order is:
+    1. if filename is set then load from it
+    2. ~/.convert2rhel.ini
+    3. /etc/convert2rhel.ini
+
+    Return dict of supported options with their values.
+    """
+    headers = ["subscription_manager"]  # supported sections in config file
+    config_file = configparser.ConfigParser()
+    paths = [os.path.expanduser(p) for p in CONFIG_PATHS]
+    # Create dict with all supported options, all of them set to None
+    # needed for avoiding problems with files priority
+    # The name of supported option MUST correspond with the name in ToolOpts()
+    # Otherwise it won't be used
+    supported_opts = {"password": None, "activation_key": None}
+
+    if cfg_path:
+        cfg_path = os.path.expanduser(cfg_path)
+        if not os.path.exists(cfg_path):
+            raise OSError(2, "No such file or directory: '" + cfg_path + "'")
+        paths.insert(0, cfg_path)  # highest priority
```

Comment:
I don't think we need this comment anymore! Since we have it in our docstrings, it should be fairly easy to spot that this is the highest priority when looking for a configuration file. 

```suggestion
        paths.insert(0, cfg_path)
```